tests/utils.py

```diff
@@ -661,3 +662,11 @@ def check_error_param_structure(body,                                   # type:
         check_val_is_in("compare", body["param"])
         if param_compare is not null:
             check_val_equal(body["param"]["compare"], param_compare)
+
+
+def check_file_permissions(file_path, permissions):
+    # type: (str, str) -> bool
```

Comment:
The `permissions` should be `int` type, and specified as `0o###` when passed as argument. 

The `file_path` could use `pathlib.Path` (or `PathLike[str]`?, I don't remember the specific name, but there is a type to indicate anything that looks like a path), to avoid the `str()` conversion each time.